docs_user/modules/proc_migrating-ceph-mds.adoc

```diff
@@ -295,8 +251,3 @@ mds.mds.cephstorage-1.jkvomp  cephstorage-1.redhat.local                     run
 mds.mds.cephstorage-2.gnfhfe  cephstorage-2.redhat.local                     running (79m)     3m ago  79m    24.2M        -  17.2.6-100.el9cp  1af7b794f353  f3cb859e2a15
 ----
 
-
-== Useful resources
```

Comment:
Wondering if it makes sense to keep these references upstream or if we can include these links (if the idea is to not have this section) in the procedure above

Reply:
Ack. I added an ifeval so it renders upstream.